commands/commands.go

```diff
@@ -55,8 +60,13 @@ func (c *Command) Run(pctx context.Context, bus *events.EventBus) {
 	log.Debugf("%s.Run start", c.Name)
 
 	cmd := ArgsToCmd(c.Exec, c.Args)
-	cmd.Stdout = c.logger.Writer()
-	cmd.Stderr = c.logger.Writer()
+	if c.logger.Logger != nil {
```

Comment:
I thought that at first but the field is `logger log.Entry` in the struct, so it'll always be zero value at worst. You can't even make the `c.logger == nil` comparison without the compiler telling you you're wrong. 😀